tempesta_fw/t/functional/wrk/head_get.lua

```diff
@@ -0,0 +1,10 @@
+init = function(args)
+        local r = {}
+        r[1] = wrk.format("HEAD", "/")
```

Comment:
As i have noticed recently in the chat, `wrk` handles responses to `HEAD` in a wrong way. It expects body in such response. That lead to very frequent connection drops by `wrk` and the responses are missed in the `wrk` statistics. I believe we should mark tests expected to fail rather than just ignore statistics errors.

Anyway `wrk` is still an only option here, may be we can prepare a patch and send it to upstream is some nearest future.

Reply:
> Anyway wrk is still an only option here, may be we can prepare a patch and send it to upstream is some nearest future.

I think, this is the right way